src/components/EditorsColumn.jsx

```diff
@@ -52,9 +52,16 @@ export default class EditorsColumn extends React.Component {
 
   _handleEditorDividerDrag(index, _, {deltaY, lastY, y}) {
     const {onUpdateFlex} = this.props;
-    const editorHeights = getNodeHeights(this.editorRefs);
-    const dividerHeights = getNodeHeights(this.dividerRefs);
-    onUpdateFlex({deltaY, dividerHeights, editorHeights, index, lastY, y});
+    const contentSizes = getNodeHeights(this.editorRefs);
+    const dividerSizes = getNodeHeights(this.dividerRefs);
+    onUpdateFlex({
+      contentSizes,
+      deltaPosition: deltaY,
+      dividerSizes,
+      index,
+      lastPosition: lastY,
+      position: y,
```

Comment:
I can do that; I do alphabetical order by habit just because it's generally uncontroversial :)